### PR TITLE
Switch to Solarized color palette

### PR DESCRIPTION
**Pull request by mizdebsk**
Currently it's hard to read Validator HTML reports due to poor selection of colors and in some cases low contrast (eg. yellow text on white background).
Instead of inventing a new color scheme, I propose to use [Solarized](https://ethanschoonover.com/solarized): a set of carefully selected colors designed for use in applications like Validator. Solarized improves readability and reduces eye fatigue.
I understand that this is a subjective matter (_de gustibus non est disputandum_), so feel free to ignore/close this PR.

### CODE REVIEW

**Comment by mkoncek** [comment]
The idea sounds good.
I have to check how the results look like myself.

**Comment by mizdebsk** [comment]
Example outputs:
- solarized: https://mizdebsk.fedorapeople.org/jpv-demo/solarized/html
- current: https://mizdebsk.fedorapeople.org/jpv-demo/current/html

**Comment by mizdebsk** [comment]
Also added the code that was used to generate the example outputs.

File: src/main/resources/tmt_html/style.css [hunk 1]
```diff
@@ -1,30 +1,50 @@
...
-.cyan           { color: darkcyan; }
-.white          { color: ghostwhite; }
-.bright.black   { color: darkgrey; }
-.bright.red     { color: red; }
-.bright.green   { color: green; }
-.bright.yellow  { color: goldenrod; }
-.bright.blue    { color: blue; }
-.bright.magenta { color: magenta; }
-.bright.cyan    { color: #00e5ff; }
-.bright.white   { color: white; }
```
Comment (mkoncek) [inline src/main/resources/tmt_html/style.css:28]: What happens if someone uses a color that was removed?
Will the renderer use a default color only?
Is there a problem assigning new color codes to colors that are currently unused?
I would prefer to have a future-proof solution.

**Comment by mkoncek** [comment]
I think it makes sense to use a generally accepted color scheme.
We can put it to test use now.

File: src/main/resources/tmt_html/style.css (hunk 1 above)
Comment (mizdebsk) [reply src/main/resources/tmt_html/style.css:28]: You mean removed bright variant? Non-bright one will be used, eg. bright green is the same as pure green.

Comment (mkoncek) [reply src/main/resources/tmt_html/style.css:28]: Ok. Sounds good.

**Pull request merged**